metadata.json

```diff
@@ -18,7 +18,7 @@
     },
     {
       "name": "icinga/icinga",
-      "version_requirement": ">= 2.9.0 < 6.0.0"
+      "version_requirement": ">= 2.9.0 < 7.0.0"
```

Comment:
I think we should do an individual PR for this, so it appears in the changelog and people recognize this was changed